license/src/com/untangle/app/license/LicenseManagerImpl.java

```diff
@@ -124,6 +134,11 @@
      */
     private boolean devLicenseTest = false;
 
+    /**
+     * Boolean for if connection is good or not
+     */
+    private boolean connectionGood = false;
```

Comment:
can this name be more meaningful? this indicates license server connectivity, so maybe something like 'ServerConnectivity' or something similar